### PR TITLE
feat: Allow ES2019 features

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: packages/tsconfig/base.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "jsx": "react",
-    "lib": ["es2016", "es2017"],
+    "lib": ["es2019"],
```
Comment (taion) [inline packages/tsconfig/base.json]: @jquense so technically node 10 doesn't have full es2019 support. but 12 is the current LTS and we should upgrade. and es(n) includes all the es(n-1)

**Comment by jquense** [comment]
seems fine to me. what specifically would this allow?

**Comment by taion** [comment]
`flatMap` :p

fixes a type error i was hitting

**Comment by jquense** [comment]
`smooshMap` tho

**Pull request merged**